website/content/docs/configuration/controller.mdx

```diff
@@ -35,18 +35,35 @@ description will be read.
     Boundary will not be able to connect by default. To run Boundary without a TLS connection
     to Postgres (not recommended for production usage), add the `sslmode=disable` parameter to
     your connection string, such as `url = "postgresql://postgres:boundary@192.168.1.1:5432/boundary?sslmode=disable"`
+    This value can refer to a file on disk (file://) from which a URL will be read; an env
+    var (env://) from which the URL will be read; or a direct database URL (postgres://).
   - `migration_url` - Can be used to specify a different URL for migrations, as that
     usually requires higher privileges.
+    This value can refer to a file on disk (file://) from which a URL will be read; an env
+    var (env://) from which the URL will be read; or a direct database URL (postgres://).
   - `max_open_connections` - Can be used to control the maximum number of
     connections that can be opened by the controller.
     The minimum number of connections required is 5.
     Setting this value to 0 will allow the controller to open as many
-    connections as needed. This value can be a string representing the max number of connections, can refer to a file
+    connections as needed. This value can be an integer representing the max number of connections, can refer to a file
```

Comment:
This went from something not quite correct to something else not quite correct :-)  This can be a string or integer in the config file which is typed; it's a string for env var or file since those are always strings. Hence string was put here. But probably it should be "string or integer"

Reply:
I reworded this and this for `max_open_connections` and `max_idle_connections`. Also I added some similar wording for `max_idle_time`. Please take another look,